src/features/student/views/StudentToolsTracesView/components/TraceAiUsageToggleFormField/TraceAiUsageToggleFormField.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+import type { CreateTraceForm, UpdateTraceForm } from '@/features/student/types'
+import { TraceAiUsageToggle } from '@/features/student/components'
+import { useAttrs } from 'vue'
+
+interface TraceAiUsageToggleFormFieldProps {
+  form: CreateTraceForm | UpdateTraceForm
+}
+
+defineProps<TraceAiUsageToggleFormFieldProps>()
+
+const emit = defineEmits<{
+  (event: 'change', payload: boolean): void
+}>()
+
+const attrs = useAttrs()
+
+function handleChange (value: boolean, fieldChange: (value: boolean) => void) {
+  fieldChange(value)
+  emit('change', value)
+}
+</script>
+
+<template>
+  <form.Field name="useIA">
+    <template #default="{ field }">
+      <div class="declaration-items__field">
+        <TraceAiUsageToggle
+          v-bind="attrs"
+          id="useIA"
+          name="useIA"
+          :model-value="field.state.value"
+          @update:model-value="(value) => handleChange(value, field.handleChange)"
+        />
+        <div
+          v-if="field.state.meta.errors.length > 0"
+          class="declaration-items__error"
+        >
+          {{ field.state.meta.errors.join(', ') }}
```

Comment:
```suggestion
          <span class="caption-regular">{{ field.state.meta.errors.join(', ') }}</span>
```
Une proposition de classe, je ne sais plus quelle est la typo pour les erreurs, faudra regarder et aussi ajouter la couleur dans le style.
Au passage, il faut aussi qu'on prenne le réflexe de toujours mettre les textes dans des span, c'est plus clean que de les mettre direct dans des divs